SimTracker/TrackTriggerAssociation/python/StubAssociator_cfi.py

```diff
@@ -0,0 +1,10 @@
+import FWCore.ParameterSet.Config as cms
+
+StubAssociator_params = cms.PSet (
+  UseTTStubAssMap         = cms.bool    ( False ),                                                   #
```

Comment:
The meaning of "UseTTStubAssMap" is not clear. A comment should make clear that either the association is taken from TTStubAssociation or from xyz, where xyz needs stating.